profile/profile.go

```diff
@@ -674,6 +674,36 @@ func numLabelsToString(numLabels map[string][]int64, numUnits map[string][]strin
 	return strings.Join(ls, " ")
 }
 
+// AddTag adds a tag with the specified key and value to all samples in the
+// profile.
+func (p *Profile) AddTag(key, value string) {
```

Comment:
I'd maybe do this as SetTag(), it would simplify the code and I don't think we really need to support multiple values per tag in this PR.

Reply:
Done.